generator/template/ready-to-use-section-types/sections/views/ResturantMenu_static.vue

```diff
@@ -0,0 +1,417 @@
+<template>
+  <div v-if="settings" class="restaurant-menu py-6" :class="settings.classes"
+    :style="{ backgroundColor: settings.backgroundColor || 'transparent' }">
+    <div class="container mx-auto px-4">
+      <!-- Restaurant Logo -->
+      <div v-if="settings.logo && settings.logo.url" class="text-center mb-6">
+        <img :src="settings.logo.url" :alt="settings.logo.seo_tag || 'Restaurant Logo'"
+          class="h-24 object-contain mx-auto" width="96" height="96" loading="lazy" />
+      </div>
+
+      <!-- Menu Title and Subtitle -->
+      <div class="text-center mb-8">
+        <h2 v-if="settings.menuTitle && settings.menuTitle[lang]" class="text-3xl font-bold mb-2">
+          {{ settings.menuTitle[lang] }}
+        </h2>
+        <p v-if="settings.menuSubtitle && settings.menuSubtitle[lang]" class="text-lg">
+          {{ settings.menuSubtitle[lang] }}
```

Comment:
The section should be neutral so no specific styles should be given ie: font size, weight color etc..

It would nice to have the title and subtitles as wysiwyg, you can check its usage in the `TextImage` section component form/view. This will give more control to the anyone using this section to style the texts